W/WCPG/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WCPG"
+version = v"0.9.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/fixif/WCPG/archive/refs/tags/0.9.tar.gz", "4f6b1d2abc298891ae9e3966428c2d8b4e8bbc3528e917d261d8613dea41ab7d"),
+    FileSource("https://www.netlib.org/clapack/f2c.h", "7d323c009951dbd40201124b9302cb21daab2d98bed3d4a56b51b48958bc76ef"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+ln -s ${libdir}/libopenblas.${dlext} ${libdir}/libblas.${dlext}
+ln -s ${libdir}/libopenblas.${dlext} ${libdir}/liblapack.${dlext}
+mv $WORKSPACE/srcdir/f2c.h $WORKSPACE/destdir/include/
```

Comment:
Unless this header file is actually needed to build downstream libraries against WCPG, there is no need to move this file to `${includedir}` (please, use this environment variable, don't use absolute paths), just add the directory where `f2c.h` to the preprocessor search path with `-I`